ChameleonForms.AcceptanceTests/Helpers/HttpClientExtensions.cs

```diff
@@ -1,140 +1,17 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
-using AngleSharp;
-using AngleSharp.Dom.Html;
-using AngleSharp.Html;
-using AngleSharp.Network;
 using ChameleonForms.AcceptanceTests.Helpers.Pages;
-using Xunit;
 
-namespace RazorPagesProject.Tests.Helpers
+namespace ChameleonForms.AcceptanceTests.Helpers
 {
     public static class HttpClientExtensions
     {
-        public static Task<HttpResponseMessage> SendAsync(
-            this HttpClient client,
-            IHtmlFormElement form,
-            IHtmlElement submitButton)
+        public static async Task<T> GetPageAsync<T>(this HttpClient client, string url) where T : IChameleonFormsPage
         {
-            return client.SendAsync(form, submitButton, new Dictionary<string, string>());
-        }
-
-        public static Task<HttpResponseMessage> SendAsync(
-            this HttpClient client,
-            IHtmlFormElement form,
-            IEnumerable<KeyValuePair<string, string>> formValues)
-        {
-            var submitElement = Assert.Single(form.QuerySelectorAll("[type=submit]"));
-            var submitButton = Assert.IsAssignableFrom<IHtmlElement>(submitElement);
-
-            return client.SendAsync(form, submitButton, formValues);
-        }
-
-        public static async Task<HttpResponseMessage> SendAsync(this HttpClient client
-            , IHtmlFormElement form
-            , IHtmlElement submitButton
-            , IEnumerable<KeyValuePair<string, string>> formValues
-            )
-        {
-            return await SendAsync(client, new[] { form }, submitButton, formValues);
-        }
-
-        public static Task<HttpResponseMessage> SendAsync(this HttpClient client

```

Comment:
All of this stuff was replicating functionality that exists within angle sharp (ability to "submit" a form and get it to determine the values to post in a http request). This meant we were by-passing some of what was going on in the form (and this actually hid some errors that were happening, which I verified by running the old acceptance tests against the new code).